app/src/test/java/com/example/parkingproject/mvp/model/ReleaseModelTest.java

```diff
@@ -0,0 +1,87 @@
+package com.example.parkingproject.mvp.model;
+
+import com.example.parkingproject.entities.Parking;
+import com.example.parkingproject.entities.Reservation;
+import com.example.parkingproject.exceptions.ReleaseMoreThanOneMatchException;
+import com.example.parkingproject.exceptions.ReleaseNoMatchException;
+import com.example.parkingproject.exceptions.ReservationListEmptyException;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+
+import java.util.Date;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+@RunWith(RobolectricTestRunner.class)
```

Comment:
Why Robolectric?

Reply:
Becouse one of the test fail if i dont use roboletric but the problem is a catch with a log.e() in a method. 
I removed the log and work fine without roboelectric